File: src/main/java/ai/aimachineserver/domain/games/tictactoe/PlayerHuman.java
package ai.aimachineserver.domain.games.tictactoe;

public class PlayerHuman extends Player {
    PlayerHuman(String name, Symbol symbol) {
        super(name, symbol);
    }

    @Override
    public void makeMove(Board board, int rowIndex, int colIndex) {
        board.setFieldValue(rowIndex, colIndex, symbol.token);
    }
}

File: src/main/kotlin/ai/aimachineserver/domain/games/tictactoe/PlayerHuman.kt
package ai.aimachineserver.domain.games.tictactoe

class PlayerHuman(
    name: String,
    symbol: Symbol
) : Player(name, symbol) {
    override fun makeMove(board: Board, rowIndex: Int, colIndex: Int) {
        board.setFieldValue(rowIndex, colIndex, symbol.token)
    }
}
